GenumKit/Parsers/StringsFileParser.swift

```diff
@@ -17,12 +31,41 @@ public final class StringsFileParser {
 
   // Localizable.strings files are generally UTF16, not UTF8!
   public func parseStringsFile(path: String) throws {
+    guard let data = NSData(contentsOfFile: path) else {
+      throw StringsFileParserError.FailureOnLoading(path: path)
+    }
+
+    let plist = try NSPropertyListSerialization.propertyListWithData(data, options: NSPropertyListReadOptions.Immutable, format: nil)
```

Comment:
Ah indeed. It's very unlikely (if even possible) that a valid Localizable.strings generates an invalid PLIST. And an invalid Localizable.strings generally makes Xcode generate a warning at compile time iirc. But I appreciate the idea of having a more explicit error so maybe you're right, that's a good idea to keep it that way